salt/modules/win_system.py

```diff
@@ -792,14 +793,44 @@ def get_domain_workgroup():
             return {'Workgroup': computer.Domain}
 
 
+def _try_parse_datetime(time_str, fmts):
+    '''
+    Attempts to parse the input time_str as a date.
+
+    :param str time_str: A string representing the time
+    :param list fmts: A list of date format strings
+
+    :return: Returns a datetime object if parsed properly. Otherwise None
+    :rtype datetime
+    '''
+    result = None
+    for fmt in fmts:
+        try:
+            result = datetime.strptime(time_str, fmt)
+            break
+        except ValueError:
+            pass
+    return result
+
+
 def get_system_time():
     '''
     Get the system time.
 
-    :return: Returns the system time in HH:MM AM/PM format.
+    :return: Returns the system time in HH:MM:SS AM/PM format.
     :rtype: str
     '''
-    return datetime.strftime(datetime.now(), "%I:%M %p")
```

Comment:
Why is this being replaced? Are you suggesting there is a bug in python's `strftime()`?